Rakefile

```diff
@@ -1,12 +1,28 @@
 require 'rake'
 require 'rspec/core/rake_task'
 require 'puppetlabs_spec_helper/rake_tasks'
+require 'puppet'
+
 begin
   require 'beaker/tasks/test' unless RUBY_PLATFORM =~ /win32/
 rescue LoadError
   #Do nothing, only installed with system_tests group 
 end
 
+
+# If puppet does not support symlinks (e.g., puppet <= 3.5) we cannot use
+# puppetlabs_spec_helper's `rake spec` task because it requires symlink
+# support. Redine `rake spec` without the symlink call and restrict the pattern
```

Comment:
Thanks! Had a commit staged and forgot to push. Will fixup once we've resolved all comments.